subworkflows/MODEL_TO_DATA.nf

```diff
@@ -33,13 +33,13 @@ include { UPDATE_SUBMISSION_STATUS as UPDATE_SUBMISSION_STATUS_AFTER_RUN } from
 include { UPDATE_SUBMISSION_STATUS as UPDATE_SUBMISSION_STATUS_AFTER_VALIDATE } from '../modules/update_submission_status.nf'
 include { UPDATE_SUBMISSION_STATUS as UPDATE_SUBMISSION_STATUS_AFTER_SCORE } from '../modules/update_submission_status.nf'
 include { VALIDATE } from '../modules/validate.nf'
-include { SCORE } from '../modules/score.nf'
+include { SCORE_MODEL_TO_DATA as SCORE } from '../modules/score_model_to_data.nf'
 include { ANNOTATE_SUBMISSION as ANNOTATE_SUBMISSION_AFTER_VALIDATE } from '../modules/annotate_submission.nf'
 include { ANNOTATE_SUBMISSION as ANNOTATE_SUBMISSION_AFTER_SCORE } from '../modules/annotate_submission.nf'
 include { SEND_EMAIL } from '../modules/send_email.nf'
 
 workflow MODEL_TO_DATA {
-    SYNAPSE_STAGE(params.input_id)
+    SYNAPSE_STAGE(params.input_id, "input")
```

Comment:
I don't think this is necessary for the PR but maybe later down the line the `input_folder_name` can be a user-fed parameter up at the top.

Reply:
I agree. I did it this way because `input/` is already hard-coded into `RUN_DOCKER`. If there is a need to be able to configure this in the future it will be easy to implement.